.github/workflows/scripts/pr-get-job-link.js

```diff
@@ -1,9 +1,9 @@
 
-module.exports = async ({ github, context, core }) => {
+module.exports = async ({ name, github, context, core }) => {
     const { owner, repo } = context.repo;
     const url = `https://api.github.com/repos/${owner}/${repo}/actions/runs/${context.runId}/jobs`
-    const result = await github.request(url)
-    const link = `https://github.com/grafana/grafana/runs/${result.data.jobs[0].id}?check_suite_focus=true`;
-
-    core.setOutput('link', link);
+    const result = await github.request(url);
+    const job = result.jobs.find(j => j.name === name);
+    
+    core.setOutput('link', job.html_url);
```

Comment:
```suggestion
    core.setOutput('link', `${job.html_url}?check_suite_focus=true`);
```

maybe to keep it the same way it was before? I think it changes the sidebar a bit / makes it more compact.

Reply:
For me that one doesn't change the UI but I'm happy to add it if needed.

Reply:
Here is an example with and without (look at the sidebar on the left):
- https://github.com/grafana/grafana/runs/10088512148?check_suite_focus=true#step:7:1 (with)
- https://github.com/grafana/grafana/runs/10088512148?#step:7:1 (without)